### PR TITLE
Add documentation on customizing the Orbit STF

### PR DESCRIPTION
**Pull request by PlasmaPower**

### CODE REVIEW

File: arbitrum-docs/launch-orbit-chain/how-tos/customize-stf.mdx [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+  Nodes (and the fraud proofs) will simply accept the new transaction ordering as there is no single ordering they think is correct.
+
+## Modification Compatibility with Arbitrum Nitro
+
+Some potential modifications are incompatible with Arbitrum Nitro and would not result in a functioning blockchain.
+Here are some requirements for the Arbitrum Nitro State Transition Function:
+
+- The STF must be deterministic. For instance, if you gave an address a random balance using the Go randomness library,
+  every node would disagree on the correct amount of balance and the blockchain would not function correctly.
+  However, it is acceptable to take a non-deterministic path to a deterministic output.
```
Comment (hkalodner) [inline arbitrum-docs/launch-orbit-chain/how-tos/customize-stf.mdx:46]: Pedantic point, but this made me think of the non-deterministic preimages issue caused by iteration over a map in the go-ethereum code that we solved :D

**Review by hkalodner — COMMENTED** [review]
Technical content all looks solidly covered to me

File: arbitrum-docs/launch-orbit-chain/how-tos/customize-precompile.mdx [hunk 2]
```diff
@@ -51,13 +51,7 @@ Then, open the corresponding Solidity interface file (`ArbSys.sol`) from the [pr
...
 ```
 
-Next, build Nitro by following steps 3-7 of the instructions in [How to build Nitro locally](/node-running/how-tos/build-nitro-locally). Note that if you've already built the Docker image, you still need run the last step to rebuild.
-
-Run Nitro with the following command:
-
-```shell
-docker run --rm -it  -v /some/local/dir/arbitrum:/home/user/.arbitrum -p 0.0.0.0:8547:8547 -p 0.0.0.0:8548:8548 @latestNitroNodeImage@ --parent-chain.connection.url=<YourParentChainUrl> --chain.id=<YourOrbitChainId> --http.api=net,web3,eth,debug --http.corsdomain=* --http.addr=0.0.0.0 --http.vhosts=*
-```
+Next, follow the steps in [How to customize your Orbit chain's behavior](./customize-stf.mdx) to build a modified Arbitrum Nitro node docker image and run it.
```
Comment (Jason-W123) [inline arbitrum-docs/launch-orbit-chain/how-tos/customize-precompile.mdx:54]: How about we redirect to ./customize-stf.mdx#Building-a-Dev-Image ?

File: arbitrum-docs/launch-orbit-chain/how-tos/customize-precompile.mdx [hunk 3]
```diff
@@ -132,13 +126,7 @@ interface ArbHi {
...
 ```
 
-Next, build Nitro by following the instructions in [How to build Nitro locally](/node-running/how-tos/build-nitro-locally). Note that if you've already built the Docker image, you still need run the last step to rebuild.
-
-Run Nitro with the following command:
-
-```shell
-docker run --rm -it  -v /some/local/dir/arbitrum:/home/user/.arbitrum -p 0.0.0.0:8547:8547 -p 0.0.0.0:8548:8548 @latestNitroNodeImage@ --parent-chain.connection.url=<YourParentChainUrl> --chain.id=<YourOrbitChainId> --http.api=net,web3,eth,debug --http.corsdomain=* --http.addr=0.0.0.0 --http.vhosts=*
-```
+Next, follow the steps in [How to customize your Orbit chain's behavior](./customize-stf.mdx) to build a modified Arbitrum Nitro node docker image and run it.
```
Comment (Jason-W123) [inline arbitrum-docs/launch-orbit-chain/how-tos/customize-precompile.mdx:129]: Same as above

**Review by Jason-W123 — APPROVED** [review]

**Review by symbolpunk — APPROVED** [review]

**Pull request merged**